app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,153 @@
+<resources>
+    -->
+    <string name="app_name">Mentoring Systers</string>
+    <string name="logo">Logo</string>
+    <string name="login">Zaloguj się</string>
+    <string name="update_profile">Edytuj profil</string>
+    <string name="sign_up">Zarejestruj się</string>
+    <string name="logging_in">Logowanie…</string>
+    <string name="signing_up">Rejestracja…</string>
+    <string name="fetching_users">Pobieranie użytkowników…</string>
+    <string name="fetch_user_profile">Pobieranie profil użytkownika…</string>
+    <string name="logging_successful">Logowanie powiodło się</string>
+    <string name="registration_successful">Pomyślnie zarejestrowano. Proszę potwierdzić swój email.</string>
+    <string name="update_successful">Aktualizacja profilu powiodła się!</string>
+    <string name="username_or_email">Nazwa użytkownika lub email</string>
+    <string name="name">Nazwa</string>
+    <string name="email">E-mail</string>
```

Comment:
Czasem email czasem e-mail, przydało by nie troche konsekwencji.

Reply:
: O
Po przeczytaniu tego komentarza dotarło do mnie, że jesteś z Polski.
Jasne, masz rację, poprawię te tłumaczenia.
---

I realized that you're from Poland after reading this comment.
Sure, I'll fix this.

Reply:
No tak 😁